proto/server.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package shortener;
```

Comment:
Please use the correct name.
https://github.com/grpc/grpc-go/blob/master/examples/features/proto/echo/echo.proto